samples/BenchmarkDotNet.Samples/Program.cs

```diff
@@ -1,9 +1,22 @@
 ﻿using System.Reflection;
 using BenchmarkDotNet.Running;
 
+#if UAP
+namespace System.Threading
+{
+    internal static class Thread
+    {
+        public static void Sleep(int milliseconds)
```

Comment:
I am not sure about the purpose of this class, but in general Thread.Sleep != Task.Delay so the name is wrong.